### PR TITLE
Fix other builds by pinning rest client

### PR DESCRIPTION
**Pull request by JonRowe**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -12,6 +12,9 @@ group :documentation do
   gem 'redcarpet',     '2.3.0'
   gem 'github-markup', '1.0.0'
   gem 'relish'
+  if RUBY_VERSION '< 2.0.0'
```
Comment (myronmarston) [inline Gemfile]: This seems to have broken things worse.  Every entry in the build matrix is failing during the `bundle install` phase with:

```
Undefined local variable or method `RUBY_VERSION' for Gemfile
```

Comment (JonRowe) [inline Gemfile]: It seems I typo'd this

**Comment by myronmarston** [comment]
These changes look fine, although were you expecting them to fix 1.8.7?

**Comment by JonRowe** [comment]
Yeah I did, but bundler is doing something weird. The conflict its complaining about doesn't exist :/

**Comment by JonRowe** [comment]
@myronmarston this is now failing for the usual reasons on 1.8.7, and so should fix the other builds (huzzah :/) turns out theres a bundler issue with displaying dependency resolution on 1.14, 1.13 enabled me to find out what the issue was.

**Comment by myronmarston** [comment]
LGTM.  Do you want to merge this and then kick the builds of the other repos to see if they are indeed green?

**Pull request merged**